adhocracy-plus/assets/scss/components/_live_questions.scss

```diff
@@ -4,3 +4,7 @@
     margin-top: -0.75*$spacer;
     margin-bottom: $spacer;
 }
+
+.live_questions__text {
+    overflow-wrap: break-word;
```

Comment:
Nice! I think else where we use "word-break: break-word", but this one seems to be better (compatible to earlier browser versions)!